.github/workflows/build-zwallet.yaml

```diff
@@ -66,6 +66,8 @@ jobs:
           echo "TAG=v0.0.0-master" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
         } ElseIf ( "${{github.base_ref}}" -eq "dev" -OR "${{github.ref}}" -eq "refs/heads/dev" ){
           echo "TAG=v0.0.0-dev" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
+        } ElseIf ( "${{github.base_ref}}" -eq "staging" -OR "${{github.ref}}" -eq "refs/heads/staging" ){
+          echo "TAG=v0.0.0-staging" | Out-File -FilePath $Env:GITHUB_ENV -Encoding utf-8 -Append
```

Comment:
does this say that the staging latest tag will no longer be called "staging"? what's the reason for the change?

Reply:
TAG is for release page. if it is staging , you will see the duplicated staging issue on https://github.com/0chain/zwalletcli/releases/tag/staging

Reply:
ah yes i remember that issue now, good fix